evaluation.py

```diff
@@ -31,31 +49,30 @@ def _vstack_images(self, img_list, interpolation=cv2.INTER_CUBIC):
         return cv2.vconcat(im_list_resize) 
 
     # Returns np array of predicted heatmaps for a given image and model
-    def predict_heatmaps(self, h, X):
-        h._load_model(self.model_json, self.weights)
-
-        X = np.expand_dims(X, axis=0) # add "batch" dimension of 1 because predict needs shape (1, 256, 256, 3)
-        predict_heatmaps = h.model.predict(X)
-        predict_heatmaps = np.array(predict_heatmaps) # output shape is (num_hg_blocks, 1, 64, 64, 17)
-
-        print('model prediction metrics: ')
-        predict_mean = np.mean(predict_heatmaps)
-        predict_max = np.max(predict_heatmaps)
-        predict_min = np.min(predict_heatmaps)
-        predict_var = np.var(predict_heatmaps.flatten())
-        print('Mean: {:0.6e}\t Max: {:e}\t Min: {:e}\t Variance: {:e}'.format(predict_mean, predict_max, predict_min, predict_var))
-        normalized_heatmaps = predict_heatmaps / predict_max
-        normalized_heatmaps = normalized_heatmaps - predict_min
-        normalized_heatmaps = predict_heatmaps / np.max(predict_heatmaps)
-
-        return normalized_heatmaps
+    def predict_heatmaps(self, X_batch):
+        predicted_heatmaps = np.array(self.model.predict(X_batch)) # output shape is (num_hg_blocks, X_batch_size, 64, 64, 17)
+        for i in range(len(X_batch)):
+            hm = predicted_heatmaps[:,i,]
+            print('model prediction metrics: ')
+            predict_mean = np.mean(hm)
+            predict_max = np.max(hm)
+            predict_min = np.min(hm)
+            predict_var = np.var(hm.flatten())
+            print('Mean: {:0.6e}\t Max: {:e}\t Min: {:e}\t Variance: {:e}'.format(predict_mean, predict_max, predict_min, predict_var))
```

Comment:
I know I added this, but I'd remove for now. It ended up cluttering up my console and slowing down the code